dummy_usb_cam_publisher:
  ros__parameters:
    camera_intrinsics:
      fx: 495.231697
      fy: 494.590030
      cx: 303.808212
      cy: 249.027693
    distortion:
      d1:  0.077718
      d2: -0.127016
      d3: -0.000550
      d4: -0.001023
      d5: 0.000000
    image_width: 640
    image_height: 480
    fps: 10
    camera_frame_id: "fingerprint_camera"
    camera_topic: "/dummy/visual/fingerprint_camera"